app/waf/src/models/adc.model.ts

```diff
@@ -220,6 +220,20 @@ export class Adc extends CommonEntity {
   constructor(data?: Partial<Adc>) {
     super(data);
   }
+
```

Comment:
It's not a good way to make a model to have behaviors, that may be a bad implementation design. 
Loopback4 is a MVC framework. Model is the place to map database layout. 

The similar situation happens to application.model.ts and common.model.ts. getAS3Declaration staff.

Reply:
Those two are property made by some calculation. This is a very common case. An object's can have some properties originate from the calculation with some other properties. I do not think moving those code can benefit the program.

Reply:
I don't think so. However, it's up to yo.